src/pages/Layout.tsx

```diff
@@ -25,20 +25,33 @@ function Layout({ darkMode, toggleDarkMode }: { darkMode: boolean, toggleDarkMod
             color: "navbar.light",
         },
     };
-
+    
+    const BurgerButtons = {
+        color: 'primary.light',
+        borderRadius: '5px',
+        padding: '1%',
+        width: '100%',
+        '&:hover': {
+            backgroundColor: "navbar.dark",
+            color: "navbar.light",
+        },
+    };
     const drawerItems = (
-        <List>
-            <ListItem component={Link} to="/" onClick={handleDrawerToggle}>
-                <ListItemText primary="Main page" />
+        <List sx={{backgroundColor:"navbar.main", height:"100%",}}>
+            <ListItem sx={{}} component={Link} to="/" onClick={handleDrawerToggle}>
+                <ListItemText sx={BurgerButtons} primary="Main page" />
+            </ListItem>
+            <ListItem component={Link} to="/about-us" onClick={handleDrawerToggle}>
```

Comment:
what do you thing about moving `/about-us` and other routes to some routes util as constants?